[2024-04-26T07:08:12.932+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-26T07:08:12.950+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Crypto_ETL.concat_datasets manual__2024-04-26T07:04:31.191724+00:00 [queued]>
[2024-04-26T07:08:12.957+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Crypto_ETL.concat_datasets manual__2024-04-26T07:04:31.191724+00:00 [queued]>
[2024-04-26T07:08:12.958+0000] {taskinstance.py:2303} INFO - Starting attempt 2 of 2
[2024-04-26T07:08:12.970+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): concat_datasets> on 2024-04-26 07:04:31.191724+00:00
[2024-04-26T07:08:12.977+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=5981) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-26T07:08:12.979+0000] {standard_task_runner.py:63} INFO - Started process 5983 to run task
[2024-04-26T07:08:12.979+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Crypto_ETL', 'concat_datasets', 'manual__2024-04-26T07:04:31.191724+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/processing_ETL.py', '--cfg-path', '/tmp/tmpyncj3pc7']
[2024-04-26T07:08:12.980+0000] {standard_task_runner.py:91} INFO - Job 33: Subtask concat_datasets
[2024-04-26T07:08:13.017+0000] {task_command.py:426} INFO - Running <TaskInstance: Crypto_ETL.concat_datasets manual__2024-04-26T07:04:31.191724+00:00 [running]> on host e30f8265f0c9
[2024-04-26T07:08:13.088+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Crypto_ETL' AIRFLOW_CTX_TASK_ID='concat_datasets' AIRFLOW_CTX_EXECUTION_DATE='2024-04-26T07:04:31.191724+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-26T07:04:31.191724+00:00'
[2024-04-26T07:08:13.090+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-26T07:08:13.128+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-04-26T07:08:13.129+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-26T07:08:13.136+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=Crypto_ETL, task_id=concat_datasets, execution_date=20240426T070431, start_date=20240426T070812, end_date=20240426T070813
[2024-04-26T07:08:13.153+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-26T07:08:13.167+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-26T07:08:13.169+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
